### DISCUSSION TITLE
Tower Defense - Project retrospective

### DISCUSSION THREAD

**Discussion by Freaky2164** [Freaky2164/TowerDefense#53] [General]
Hi all,

We did a retrospective for our project. Our results you can find in the image below.

![image](https://github.com/Freaky2164/TowerDefense/assets/97874901/e2f73ca5-7913-4ba0-bfa7-8940d5a39461)

# Continue
On continuing, we summed up what helped us during our project. The main points here are our regular team meetings, which were very helpful in keeping us all up to date. We also wanted to mention Chat GPT because we sometimes used it to find shortcuts or easier ways to create or build things in Unity. Together, we made an effort every week to motivate each other, which was also a very important part of staying productive and avoiding laziness.

# Stop
Here we identified what held us back. Of course, we have to mention the cost and licenses that hindered us in our CI/CD (Continuous Integration/Continuous Deployment). We also noticed in the past months that it wasn't always easy to apply the project management techniques from our lecture to game development. This sometimes led to spending a lot of time discussing and clarifying tasks. At the beginning of the project, we were sometimes confused about Unity and its behavior, so we needed to invest extra time in getting to know Unity, which is also listed here.

# Invent
Here we mentioned things we could do differently in the future. For example, we sometimes misunderstood each other, which led to confusing situations. To address this, we want to be more precise in our communication when discussing tasks or specific changes. Additionally, we realized that we spent too much time on creating the game's design. While important, it held us back from developing new features and game mechanics.

# Act
Here we listed things we should do next. As we have already done and will continue to do in the future, we need to improve the cleanliness of our code to save time when someone else is working on it. Additionally, we are reducing the scope of our project. Initially, we had a large game in mind, but it has ended up being much smaller. To create a larger game, we would have needed more time available. 

Greetings
TD Team

**Comment by MikoHeuken** [comment]
Hey Team Tower Defense,

like always, nice blog. I really like the style of the graphic, it´s really simple and the colors fit well.
The next big plus is that you summarized the entire graphic to the four paragraphs.

Keep on going.
Team URH
